index.rst

```diff
@@ -672,43 +701,61 @@ REST API
 To facilitate debugging of prepuller issues, there is a read-only REST API to see the status of a prepull configuration.
 Changing the configuration requires changing the Helm chart or the generated ``ConfigMap`` object and restarting the lab controller.
 
-All of these API calls require ``admin:notebook`` scope.
+All of these API calls require ``admin:jupyterlab`` scope.
 
 ``GET /nublado/spawner/v1/images``
-    Returns a list of known images and their names.
+    Returns an object consisting of the images allowed by the bot users,
+    followed by a key ``all`` whose contents are all known images and
+    their names.  Each of the top-level object keys is the string
+    literal exactly as shown.
     This parses the available images according to SQR-059_ and shows the results:
 
     .. code-block:: json
 
        {
            "recommended": {
-               "url": "<full image url>",
-               "tag": "<image tag>",
+               "path": "<full Docker image path>",
+               "tags": {
+                   "<image tag>": "<human readable name>"
+               },
```

Comment:
I know that images can have multiple tags, but I don't like returning all of them to the user. I think we should pick one and provide that tag and description, and if there are aliases, whatever. In general it should be the non-alias tag.

Reply:
I find debugging to be easier if I can look at the return and quickly see which images are aliased, but I'm not wedded to the idea.

Reply:
Oh, I see. Let's go ahead and merge this for now although I think maybe just a list of aliases would be as useful (not sure the human-readable names of all of the aliases will add much). But I can propose that separately and we can talk about that then.